### PR TITLE
[NUI] Change property get/set not to use PropertyValue (patch set5)

### PR DESCRIPTION
**Pull request by hinohie**
* This should be merged with : https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/283196/
 * NUI_VISUAL_PROPERTY_CHANGE_X this macro will be deleted after all changes are completed.
 * TODO : local test not been done. only Ubuntu tested

Signed-off-by: Eunki, Hong <eunkiki.hong@samsung.com>

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 1]
```diff
@@ -1349,9 +1349,13 @@ public Vector2 ScreenPosition
             get
             {
                 Vector2 temp = new Vector2(0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_1
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:1352]: thank you for the patch.
but could you add `private Vector2 internalScreenPosition` as View's member and use it.
and in View's Dispose(), make it disposed.
could you please refer to these? 
https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/View.cs#L89
https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs#L1526
thank you.

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 2]
```diff
@@ -1737,9 +1749,13 @@ public Vector3 WorldPosition
             get
             {
                 Vector3 temp = new Vector3(0.0f, 0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_3
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:1752]: same as above

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 3]
```diff
@@ -1910,9 +1926,13 @@ public Vector3 WorldScale
             get
             {
                 Vector3 temp = new Vector3(0.0f, 0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_1
+                Object.InternalRetrievingPropertyVector3(SwigCPtr, View.Property.WorldScale, temp.SwigCPtr);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:1930]: same as above

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 4]
```diff
@@ -1954,9 +1978,13 @@ public Vector4 WorldColor
             get
             {
                 Vector4 temp = new Vector4(0.0f, 0.0f, 0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_3
+                Object.InternalRetrievingPropertyVector4(SwigCPtr, View.Property.WorldColor, temp.SwigCPtr);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:1982]: same as above

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 5]
```diff
@@ -2419,18 +2447,26 @@ private Position InternalAnchorPoint
             get
             {
                 Position temp = new Position(0.0f, 0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_1
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:2450]: same as above

**Review by dongsug-song — APPROVED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/View.cs (hunk 1 above)
Comment (hinohie) [reply src/Tizen.NUI/src/public/BaseComponents/View.cs]: Fix done. I make `internalCurrentScreenPosition` and get in on `ViewInternal.cs`.

File: src/Tizen.NUI/src/public/BaseComponents/View.cs (hunk 3 above)
Comment (hinohie) [reply src/Tizen.NUI/src/public/BaseComponents/View.cs]: I found that `WorldScale` is exactly equal with `CurrentWorldScale` intenrally. So I make it `ViewInternal.cs` API, which we already changed.

File: src/Tizen.NUI/src/public/BaseComponents/View.cs (hunk 4 above)
Comment (hinohie) [reply src/Tizen.NUI/src/public/BaseComponents/View.cs]: I found that `WorldColor` is exactly equal with `CurrentWorldColor` intenrally. So I make it `ViewInternal.cs` API, which we already changed.

File: src/Tizen.NUI/src/public/BaseComponents/View.cs (hunk 5 above)
Comment (hinohie) [reply src/Tizen.NUI/src/public/BaseComponents/View.cs]: I check that we already have `AnchorPoint` relative API internal. and they use Vector3.

`ParentOrigin` use `Position`type so I also change it use `Position` type, and also use `ViewInternal.cs` API.

**Pull request merged**